<!-- -*- mode: XML -*- -->
<!-- run object_detection/ObjectDetectionNodelet in a nodelet manager -->

<launch>

    <include file="$(find object_detection)/launch/object_detection.launch">
        <arg name="start_as_nodelet" value="true"/>

        <arg name="input_topic" value="/laser_segmenter_objects"/>
        <arg name="fixed_frame" value="world"/>

        <arg name="certainty_thresh" value="0.5" />

        <arg name="cluster_tolerance" value="1.0" />
        <arg name="min_cluster_size" value="2" />
        <arg name="max_cluster_size" value="25000" />

        <arg name="min_object_height" value="0.01" />
        <arg name="max_object_height" value="0.33" />
        <arg name="max_object_width" value="0.36" />
        <arg name="max_object_altitude" value="0.45" />
    </include>

</launch>
